### ISSUE TITLE
`bbrf $operation --force` flag

### ISSUE THREAD

**Issue by Regala** [honoki/bbrf-client#86]
Hi! This would be a simple flag to force/override operation on a document. An example would be you are trying to tag or adding a url. However, it is not in scope for the current program (maybe the scope changed in the meantime). However you'd like to force add, regardless of the scope or other non breaking validations that might exist. This would add it no matter what.

**Comment by Regala** [comment]
Another example:

```
bbrf inscope remove lots.gems.org -p ibp
WARNING! This will remove 0 document(s) from your datastore and cannot be reverted.
Are you sure you want to continue? [y/N] n
```

By using e.g. 

`bbrf inscope remove rubygems.org -p ibb --force` the document would be forcefully removed

**Comment by pdelteil** [comment]
Hi there,

You could achieve the same using `echo`:

``echo y | bbrf inscope remove rubygems.org -p ibb``

**Comment by Regala** [comment]
Bash. The more you know 🥲

Thanks!

**Comment by honoki** [comment]
Implemented in #[ffa4bc7](https://github.com/honoki/bbrf-client/commit/ffa4bc79497912b2319003dc456646d58fd87555)
